### ISSUE TITLE
Accessibility Issues in Product Page

### ISSUE THREAD

**Issue by nirajCITZ** [bcgov/api-services-portal#794]
1)
Description:Ensures the contrast between foreground and background colors meets WCAG 2 AAA enhanced contrast ratio thresholds
Helpurl:      https://dequeuniversity.com/rules/axe/4.7/color-contrast-enhanced?application=axeAPI

[
    {
        "any": [
            {
                "id": "color-contrast-enhanced",
                "data": {
                    "fgColor": "#1a5a96",
                    "bgColor": "#f1f1f1",
                    "contrastRatio": 6.3,
                    "fontSize": "10.5pt (14px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "7:1"
                },
                "relatedNodes": [
                    {
                        "html": "<body class=\"chakra-ui-light\">",
                        "target": [
                            "body"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 6.3 (foreground color: #1a5a96, background color: #f1f1f1, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 7:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<a class=\"chakra-breadcrumb__link css-5yp7oh\" href=\"/manager/namespaces\">Namespaces (newplatform)</a>",
        "target": [
            ".css-5yp7oh"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 6.3 (foreground color: #1a5a96, background color: #f1f1f1, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 7:1"
    },
    {
        "any": [
            {
                "id": "color-contrast-enhanced",
                "data": {
                    "fgColor": "#636363",
                    "bgColor": "#fcfcfc",
                    "contrastRatio": 5.85,
                    "fontSize": "10.5pt (14px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "7:1"
                },
                "relatedNodes": [
                    {
                        "html": "<section class=\"chakra-modal__content css-9fvy5c\" role=\"dialog\" id=\"chakra-modal-24\" tabindex=\"-1\" aria-modal=\"true\" aria-labelledby=\"chakra-modal--header-24\" aria-describedby=\"chakra-modal--body-24\" style=\"opacity: 0.978727; transform: scale(0.998936) translateZ(0px);\">",
                        "target": [
                            "#chakra-modal-24"
                        ]
                    },
                    {
                        "html": "<div class=\"chakra-modal__overlay css-hdd9l7\" style=\"opacity: 1;\"></div>",
                        "target": [
                            ".chakra-modal__overlay"
                        ]
                    },
                    {
                        "html": "<body class=\"chakra-ui-light\">",
                        "target": [
                            "body"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 5.85 (foreground color: #636363, background color: #fcfcfc, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 7:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<div id=\"field-61-helptext\" class=\"chakra-form__helper-text css-1y1zyf5\">Select the first environment for this product</div>",
        "target": [
            "#field-61-helptext"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 5.85 (foreground color: #636363, background color: #fcfcfc, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 7:1"
    }
]

![image.png](https://images.zenhubusercontent.com/61a121c4a7a3264dfdc530ef/bfc852d7-a16d-4f01-85ae-ba3c25dc8608)
------------------------------------------------------------------------------------------------------------
2)
Description:  Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds
Helpurl:      https://dequeuniversity.com/rules/axe/4.7/color-contrast?application=axeAPI
[
    {
        "any": [
            {
                "id": "color-contrast",
                "data": {
                    "fgColor": "#b0b0b0",
                    "bgColor": "#ffffff",
                    "contrastRatio": 2.16,
                    "fontSize": "12.0pt (16px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "4.5:1"
                },
                "relatedNodes": [
                    {
                        "html": "<div class=\"css-2e3m9t\">",
                        "target": [
                            ".css-2e3m9t"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 2.16 (foreground color: #b0b0b0, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<em class=\"chakra-text css-1t28pyd\">No services yet</em>",
        "target": [
            "em"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 2.16 (foreground color: #b0b0b0, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
    }
]
![image.png](https://images.zenhubusercontent.com/61a121c4a7a3264dfdc530ef/15c61669-8b84-46a1-85c9-531c5c300674)
------------------------------------------------------------------------------------------
3)Description:  Ensures every id attribute value of active elements is unique
Helpurl:      https://dequeuniversity.com/rules/axe/4.7/duplicate-id-active?application=axeAPI
[
    {
        "any": [
            {
                "id": "duplicate-id-active",
                "data": "field-61",
                "relatedNodes": [
                    {
                        "html": "<input class=\"chakra-radio__input\" id=\"field-61\" required=\"\" aria-required=\"true\" type=\"radio\" name=\"environment\" value=\"test\" aria-describedby=\"field-61-helptext\" style=\"border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;\">",
                        "target": [
                            ".chakra-radio__input[value=\"test\"][type=\"radio\"]"
                        ]
                    },
                    {
                        "html": "<input class=\"chakra-radio__input\" id=\"field-61\" required=\"\" aria-required=\"true\" type=\"radio\" name=\"environment\" value=\"sandbox\" aria-describedby=\"field-61-helptext\" style=\"border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;\">",
                        "target": [
                            ".chakra-radio__input[value=\"sandbox\"][type=\"radio\"]"
                        ]
                    },
                    {
                        "html": "<input class=\"chakra-radio__input\" id=\"field-61\" required=\"\" aria-required=\"true\" type=\"radio\" name=\"environment\" value=\"prod\" aria-describedby=\"field-61-helptext\" style=\"border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;\">",
                        "target": [
                            "input[value=\"prod\"]"
                        ]
                    },
                    {
                        "html": "<input class=\"chakra-radio__input\" id=\"field-61\" required=\"\" aria-required=\"true\" type=\"radio\" name=\"environment\" value=\"other\" aria-describedby=\"field-61-helptext\" style=\"border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;\">",
                        "target": [
                            ".chakra-radio__input[value=\"other\"][type=\"radio\"]"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Document has active elements with the same id attribute: field-61"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<input class=\"chakra-radio__input\" id=\"field-61\" required=\"\" aria-required=\"true\" type=\"radio\" name=\"environment\" value=\"dev\" checked=\"\" aria-describedby=\"field-61-helptext\" style=\"border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;\">",
        "target": [
            ".chakra-radio__input[value=\"dev\"][type=\"radio\"]"
        ],
        "failureSummary": "Fix any of the following:\n  Document has active elements with the same id attribute: field-61"
    }
]
---------------------------------------------------------------------------------------------------
4)Description:  Ensures every HTML document has a lang attribute
Helpurl:      https://dequeuniversity.com/rules/axe/4.7/html-has-lang?application=axeAPI
[
    {
        "any": [
            {
                "id": "has-lang",
                "data": {
                    "messageKey": "noLang"
                },
                "relatedNodes": [],
                "impact": "serious",
                "message": "The <html> element does not have a lang attribute"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<html style=\"--chakra-ui-color-mode:light;\">",
        "target": [
            "html"
        ],
        "failureSummary": "Fix any of the following:\n  The <html> element does not have a lang attribute"
    }
]
-------------------------------------------------------------------------------------------------------
5)Description:  Ensures interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive technologies
Helpurl:      https://dequeuniversity.com/rules/axe/4.7/nested-interactive?application=axeAPI
[
    {
        "any": [
            {
                "id": "no-focusable-content",
                "data": null,
                "relatedNodes": [
                    {
                        "html": "<button type=\"button\" class=\"chakra-button css-mwqlde\">",
                        "target": [
                            ".css-mwqlde"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has focusable descendants"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<button type=\"button\" class=\"chakra-button chakra-menu__menu-button css-dfn8cm\" title=\"Add environment\" data-testid=\"gateway-services-api-add-env-btn\" id=\"menu-button-97\" aria-expanded=\"false\" aria-haspopup=\"menu\" aria-controls=\"menu-list-97\">",
        "target": [
            "#menu-button-97"
        ],
        "failureSummary": "Fix any of the following:\n  Element has focusable descendants"
    }
]

**Comment by veep-22** [comment]
linked to APS-2185 on JIRA https://dpdd.atlassian.net/browse/APS-2185
